podcasts/Settings.swift

```diff
@@ -849,6 +849,16 @@ class Settings: NSObject {
         }
     }
 
+    static var darkUpNextTheme: Bool {
+        get {
+            Constants.UserDefaults.appearance.darkUpNextTheme.value
+        }
+
+        set {
+            Constants.UserDefaults.appearance.darkUpNextTheme.save(newValue)
+        }
+    }
+
```

Comment:
I can see that. I put it there because I didn't want to overcrowd the Theme section, and make it harder to find the theme/app icon options for an option that may not be used much. 

We can always move it up later.